### DISCUSSION TITLE
New Experimental Question on Efficiency

### DISCUSSION THREAD

**Discussion by dstoddar** [microsoft/vsts-extension-retrospectives#470] [General]
We are evaluating another question for the Team Assessment and would like your feedback. Is the question useful? Does it help your team identify opportunities to eliminate waste? Do you have recommendations for improvement? What is the impact of this question on your team?

Efficiency: Tools/resources/processes/procedures allow me to effectively meet my customers' needs

**Comment by dstoddar** [comment]
We've added the new question and it's available for all to use! Would love your feedback!  
![image](https://user-images.githubusercontent.com/75645919/198109100-75690f4a-81f3-4478-93d1-d47089c48dd7.png)

**Reply by ghost** [reply]
How about creating unique icon, correcting punctuation in description, and dropping "experimental"?  
1) Currently Efficiency and Energy share same icon, defeating the purpose of having an icon.  
2) Also use of slash ("/") is poor grammar; instead use a comma replacing "Tools/resources/processes/procedures" with "Tools, resources, processes, and procedures".  
3) Seems like it's past time being "experimental", so might update the details in the information icon, too.

**Comment by clq** [comment]
We're finding it useful!

The survey in general has been useful to us - mainly as a function of spawning discussions around the topics in our sprint reviews. This question in particular is interesting when put up against energy: sone sprints will be "low", i.e. there will have been team-wide problems affecting the work as a whole, which presses down both energy and efficiency. But when energy is significantly higher than efficiency, that indicates that there may be more tangible problems with systems or processes that may be actionable.

**Reply by dstoddar** [reply]
Thank you for sharing your perspective! We are finding that the question roots out opportunities for eliminating waste in the system as well!!

**Comment by mihaibusita449** [comment]
Hi guys! First of all, this Retro extension is very useful for our team and we would like to thank you for making it available! 

Secondly, is it possible to customize your own team assessment form in order to tailor it better for the team's needs? Thanks!

**Reply by dstoddar** [reply]
> Hi guys! First of all, this Retro extension is very useful for our team and we would like to thank you for making it available!
> 
> Secondly, is it possible to customize your own team assessment form in order to tailor it better for the team's needs? Thanks!

Thank you for the feedback @mihaibusita449 . One of the top priorities for our backlog is adding the ability to customize the questions. Stay tuned.....

**Comment by ghost** [comment]
Two style complaints with the Efficiency score: 1) the lightening bolt icon is the same as Energy score, 2) the use of slash, instead of comma, is unnecessarily sloppy English.

I've observed some team members blurring Efficiency and Energy when discussing their overall scores, which I think is in part due to the identical icons.

Without slash would be proper English punctuation:
_Tools, resources, processes, and procedures allow me to effectively meet my customers' needs_
